state_machine/applications/system-check/main.py

```diff
@@ -1,3 +1,7 @@
+"""
+Python System Check Module for PyCubed Mini satellite board
```

Comment:
delete `Python` and `board`
Capitalize only first words and proper nouns. (check module should not be capitalized)

Reply:
Same for all subsequent files